static/app/components/modals/inviteMembersModal/inviteStatusMessage.tsx

```diff
@@ -57,12 +94,45 @@ export default function InviteStatusMessage({
       );
     }
     const inviteRequests = (
-      <strong>{tn('%s invite request', '%s invite requests', sentCount)}</strong>
+      <strong data-test-id="sent-invite-requests">
+        {tn('%s invite request', '%s invite requests', sentCount)}
+      </strong>
+    );
+    const failedInviteRequests = (
+      <strong data-test-id="failed-invite-requests">
+        {tn('%s invite request', '%s invite requests', errorCount)}
```

Comment:
how important is the `request` copy? it seems like that's adding quite a bit of replicated code, i wonder if we could either set the `'%s invite requests'` as a prop or just remove `requests` and re-use the same `tctComponents` variable.